client/main.lua

```diff
@@ -37,22 +54,41 @@ RegisterNetEvent('qb-atms:client:updateBankInformation', function(banking)
     })
 end)
 
--- qb-target
-if Config.UseTarget then
-    CreateThread(function()
-        exports['qb-target']:AddTargetModel(Config.ATMModels, {
-            options = {
-                {
-                    event = 'qb-atms:server:enteratm',
-                    type = 'server',
-                    icon = "fas fa-credit-card",
-                    label = "Use ATM",
-                },
-            },
-            distance = 1.5,
-        })
-    end)
-end
+local NewZones = {}
+CreateThread(function()
+    if not Config.UseTarget then
+        for k, v in pairs(Config.ATMcoords) do
+            NewZones[#NewZones+1] = CircleZone:Create(vector3(v.x, v.y, v.z), 1.5, {
+                useZ = true,
+                debugPoly = false,
+                name = k,
+            })
+        end
+
+        local combo = ComboZone:Create(NewZones, {name = "RandomZOneName", debugPoly = false})
+        combo:onPlayerInOut(function(isPointInside, point, zone)
+            if isPointInside then
+                exports["qb-core"]:DrawText("[E] Use ATM")
+                Listen4Control()
+            else
+                exports["qb-core"]:HideText()
+                listen = false
+            end
+        end)
+	else
+		exports['qb-target']:AddTargetModel(Config.ATMModels, {
+				options = {
+					{
+						event = 'qb-atms:server:enteratm',
+						type = 'server',
+						icon = "fas fa-credit-card",
+						label = "Use ATM",
+					},
+				},
+				distance = 1.5,
+		})
+	end
+end)
```

Comment:
```suggestion
local NewZones = {}
CreateThread(function()
    if not Config.UseTarget then
        for k, v in pairs(Config.ATMcoords) do
            NewZones[#NewZones+1] = CircleZone:Create(vector3(v.x, v.y, v.z), 1.5, {
                useZ = true,
                debugPoly = false,
                name = k,
            })
        end

        local combo = ComboZone:Create(NewZones, {name = "RandomZOneName", debugPoly = false})
        combo:onPlayerInOut(function(isPointInside, point, zone)
            if isPointInside then
                exports["qb-core"]:DrawText("[E] Use ATM")
                Listen4Control()
            else
                exports["qb-core"]:HideText()
                listen = false
            end
        end)
    else
        exports['qb-target']:AddTargetModel(Config.ATMModels, {
            options = {
                {
                    event = 'qb-atms:server:enteratm',
                    type = 'server',
                    icon = "fas fa-credit-card",
                    label = "Use ATM",
                },
            },
            distance = 1.5,
        })
    end
end)
```
Incorrect linting / spacing